sdk/src/test/java/io/opentelemetry/sdk/metrics/DoubleCounterSdkTest.java

```diff
@@ -59,33 +60,32 @@ public void collectMetrics_NoRecords() {
     DoubleCounter doubleCounter =
         testSdk
             .doubleCounterBuilder("testCounter")
-            .setConstantLabels(ImmutableMap.of("sk1", "sv1"))
+            .setConstantLabels(Collections.singletonMap("sk1", "sv1"))
             .setLabelKeys(Collections.singletonList("sk1"))
             .setDescription("My very own counter")
-            .setUnit("metric tonnes")
+            .setUnit("ms")
```

Comment:
much less fun than to think of OTel being used to measure tonnage of something. ;)